src/QirRuntime/README.md

```diff
@@ -36,26 +36,29 @@ CI builds and tests are enabled for this project. The build has no external depe
 ### Windows pre-reqs
 
 1. Install Clang, Ninja and CMake from the public distros.
-2. Add all three to your/system `%PATH%`.
-3. Install VS 2019 and enable "Desktop development with C++" component (Clang uses MSVC's standard library on Windows).
-4. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
-5. <_optional_> To use build/test scripts install Python 3.8.
+1. Add all three to your/system `%PATH%`.
+1. Install VS 2019 and enable "Desktop development with C++" component (Clang uses MSVC's standard library on Windows).
+1. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
+1. Install the same version of dotnet as specified by qsharp-runtime [README](../../README.md)
+1. <_optional_> To use build/test scripts install Python 3.8.
 
 *Building from Visual Studio and VS Code is **not** supported.
 Running cmake from the editors will likely default to MSVC or clang-cl and fail.*
 
 ### Linux via WSL pre-reqs
 
-1. On the host Windows machine [enable WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and install Ubuntu 20.04 LTS.
-2. In the Ubuntu's terminal:
+1. On the host Windows machine [enable WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and install
+ Ubuntu 20.04 LTS.
+1. In the Ubuntu's terminal:
     1. `$ sudo apt install cmake` (`$ cmake --version` should return 3.16.3)
-    2. `$ sudo apt-get install ninja-build` (`$ ninja --version` should return 1.10.0)
-    3. `$ sudo apt install clang` (`$ clang++ --version` should return 10.0.0)
-    4. Set Clang as the preferred C/C++ compiler:
+    1. `$ sudo apt-get install ninja-build` (`$ ninja --version` should return 1.10.0)
+    1. `$ sudo apt install clang` (`$ clang++ --version` should return 10.0.0)
```

Comment:
> sudo **apt-get**
sudo **apt**

Why different?

Reply:
Probably, because I've changed from manually maintained numbered points (which are a pain) to the automatic ones?